README.md

```diff
@@ -141,6 +141,6 @@ While Concepts are an integral part of most xAPI profiles, this library does not
 
 ## License
 
-Copyright © 2021 Yet Analytics
+Copyright © 2021 Yet Analytics, Inc.
```

Comment:
Can we also add "2019-2021" as well? When I changed "2019" to "2021" I wasn't aware that projects generally included the project start year here.